### DISCUSSION TITLE
Remove text indent for python entity names to simplify customization

### DISCUSSION THREAD

**Discussion by Yura52** [pradyunsg/furo#696] [Ideas] [answered]
Hi! I am trying to replicate what I did in one of my projects with a different theme, namely, the top borders for autodocumented methods. Example: [link](https://yura52.github.io/rtdl/stable/api/rtdl.FTTransformer.html#rtdl.FTTransformer.get_baseline_transformer_subconfig)

In that project, I achieved this via the following css: [link](https://github.com/Yura52/rtdl/blob/60ae9814585d9792e112608a27e2bbbbdcbd3388/docs/_static/custom.css)

However, in Furo, it looks like in the attached screenshot. Should this be fixed on Furo side or on my side? In the latter case, I would appreciate if someone points to specific properties I should change so that I don't break things on other devices.

<img width="846" alt="Screenshot 2023-08-17 at 17 48 54" src="https://github.com/pradyunsg/furo/assets/8861742/22d3866d-3f14-4518-86c8-ddcac12ab5fa">

**Comment by pradyunsg** [comment, marked as answer]
You can explicitly set `text-indent: 0` while using a CSS selector with a [higher specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) than what Furo uses. That's a no-op if Furo ever changes behaviors and drops the indent, so it should be OK. (I'm assuming you've seen the "Unstable" banner in https://pradyunsg.me/furo/customisation/injecting/)

I do have it on my TODO to revisit how Furo handle API wrapping given https://github.com/sphinx-doc/sphinx/pull/11011, but it's a really long list -- I don't wanna give you false hope that I'd get to this anytime soon; I don't know even what the "right" behaviours should be at this point. 😅

**Reply by Yura52** [reply]
To add some context: I am looking for a way to visually separate the automatically documented methods from each other in the "feed of methods" produced by the `.. autoclass::` directive with the `:members:` option.

In case it can be helpful for others, here is one more solution (which I currently use):

```
dl.py.class dd dl.py.method {
    margin-top: 2em;
}
```

😄

**Reply by Yura52** [reply]
Just in case, I will mention that there was a bug in my CSS that resulted in the above screenshot, so my initial report is not relevant anymore.
